e2e/src/bin/historic_prices.rs

```diff
@@ -5,8 +5,9 @@ use pricegraph::{Element, Pricegraph, TokenId};
 use std::{fs::File, io::Write, path::PathBuf};
 use structopt::StructOpt;
 
-/// Threshold at which a price estimate is considered "bad"; currently 10%.
-const BAD_ESTIMATE_THRESHOLD: f64 = 0.1;
+/// Threshold logarithmic distance from the actual price at which an estimate is
+/// considered "bad"; currently `0.1 * price < estimate < 10 * price`.
+const BAD_ESTIMATE_THRESHOLD_LOG_DISTANCE: f64 = 1.0;
```

Comment:
Alternatively we could use a normal factor (10 in this case) but I don't mind logarithm either.

Reply:
I just thought it was easier to write with log and we usually say "within x orders of magnitude" for estimates that the base-10 log made sense here.